actions/universal-fix-dropbox-links.js

```diff
@@ -252,20 +247,20 @@ module.exports = (course, item, callback) => {
     function getCorrectLinks(itemPropertiesArray) {
         var brokenLinks = [];
 
-        itemPropertiesArray.forEach((link, index) => {
+        itemPropertiesArray.forEach((link) => {
             //retrieve correct Dropbox link
             var newUrl = getCanvasUrl(link);
 
             //the Canvas Dropbox does not exist in the course
-            if (newUrl === '' || typeof newUrl === 'undefined') {
-                course.error('You may want to investigate this course a little bit more since a dropbox is missing.');
+            if (newUrl === null || newUrl === undefined) {
```

Comment:
This can be set to just check if `newUrl` is truthy or not. Both `null` and `undefined` are falsy.